### ISSUE TITLE
Implement basic testing strategy for Swyxkit

### ISSUE THREAD

**Issue by sw-yx** [sw-yx/swyxkit#63]
We copied over the default Playwright tests in https://github.com/sw-yx/swyxkit/issues/62

But they are super naive. perhaps we should test the navigation and the search?

help?

**Comment by taismassaro** [comment]
hey @sw-yx, I'm not familiar with Playwright, so I'd like to give this a try if it's still up for grabs :)

**Comment by sw-yx** [comment]
yes @taismassaro up for grabs! also you can talk to @debs-obrien if you need playwright help!
